app/services/outliers/calculate_distance.rb

```diff
@@ -0,0 +1,9 @@
+class Outliers::CalculateDistance
+  def call(lng_lat1, lng_lat2)
```

Comment:
What do you think about using `coordinates` as a name instead of `lng_lat`? And then we could use `longitude, latitude = coordinates` to unpack them.

Reply:
I prefer `lng_lat` cause it's clear what's the order in the array. Makes it harder to do 
```
lat, lng = lng_lat
```
which is wrong
vs
```
lng, lat = lng_lat
```